plugins/org.wso2.developerstudio.eclipse.esb.docker/resources/init.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+# ------------------------------------------------------------------------
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
```

Comment:
Correct the licence header.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/800